lib/gobstones/spec/example.rb

```diff
@@ -0,0 +1,74 @@
+module Gobstones::Spec
+  class Example
+    include Gobstones::WithTempfile
+
+    attr_reader :gobstones_path
+
+    def initialize(gobstones_path)
+      @gobstones_path = gobstones_path
+    end
+
+    def start!(source_file, initial_board, final_board)
```

Comment:
Ruby convention: if it has some kind of effect, it is a good idea to signal it that way. 

However, not every ruby code follows this convention.